<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>87</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="scanBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>56</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>56</width>
       <height>22</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>56</width>
       <height>22</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPort">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>20</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>23</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
      </font>
     </property>
     <property name="text">
      <string>Baud Rate :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBaudRate">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>20</y>
       <width>90</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>54</y>
       <width>154</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>54</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>110</width>
       <height>14</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Status : Disconnect</string>
     </property>
    </widget>
    <widget class="QLabel" name="iconStatus">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>2</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/Red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupExport">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>256</y>
      <width>125</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>114</width>
      <height>80</height>
     </size>
    </property>
    <property name="title">
     <string>Export</string>
    </property>
    <widget class="QPushButton" name="exportBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>105</width>
       <height>43</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>98</width>
       <height>35</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Save To MS
Excel </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupGraph">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>541</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Graph (Real-Time/Second)</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="content_plot" native="true">
       <property name="minimumSize">
        <size>
         <width>502</width>
         <height>461</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>343</y>
      <width>350</width>
      <height>198</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>317</width>
      <height>198</height>
     </size>
    </property>
    <property name="title">
     <string>Data Grid View (Real-Time/Second)</string>
    </property>
    <widget class="QTableWidget" name="tableData">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>20</y>
       <width>327</width>
       <height>170</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>302</width>
       <height>170</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-color: yellow;</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>VALUE</string>
      </property>
      <property name="background">
       <color>
        <red>252</red>
        <green>251</green>
        <blue>253</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TIME</string>
      </property>
      <property name="background">
       <color>
        <red>252</red>
        <green>251</green>
        <blue>253</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DATE</string>
      </property>
      <property name="background">
       <color>
        <red>252</red>
        <green>251</green>
        <blue>253</blue>
       </color>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupCharacteristics">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>350</width>
      <height>153</height>
     </rect>
    </property>
    <property name="title">
     <string>Characteristics</string>
    </property>
    <widget class="QLabel" name="pd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>120</width>
       <height>13</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Picth Diameter : Pd</string>
     </property>
    </widget>
    <widget class="QLabel" name="fi">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>76</y>
       <width>120</width>
       <height>13</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Radial Composite : Fi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="fiTot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>103</y>
       <width>120</width>
       <height>13</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Tooth to tooth : fi</string>
     </property>
    </widget>
    <widget class="QLabel" name="fr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>120</width>
       <height>13</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>Runout Error : Fr</string>
     </property>
    </widget>
    <widget class="QLabel" name="pdResult">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>22</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fiResult">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>76</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fiTotResult">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>103</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="frResult">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>130</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>0</y>
       <width>52</width>
       <height>14</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>52</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>14</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Result</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>0</y>
       <width>60</width>
       <height>14</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>14</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Tolerance</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>0</y>
       <width>52</width>
       <height>13</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Ref.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gearTeeth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>120</width>
       <height>13</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>106</width>
       <height>13</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No. of Gear teeth&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="gearTeethResult">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>50</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="frRef">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>127</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fiRef">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>73</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fiTotRef">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>100</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Waiting...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputPd">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputGearTeeth">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>45</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputFi">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>72</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_fi">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>99</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputFr">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>126</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>256</y>
      <width>215</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>197</width>
      <height>74</height>
     </size>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="startBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>21</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>20</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>21</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>20</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBtn">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>48</y>
       <width>193</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>179</width>
       <height>20</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Clear DataGridView and Graph</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>62</width>
       <height>14</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Recording</string>
     </property>
    </widget>
    <widget class="QLabel" name="recordIconStatus">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>2</y>
       <width>11</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/Red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
